robot/Cumulus/resources/OpportunityPageObject.py

```diff
@@ -24,7 +24,7 @@ def ensure_opportunity_details_are_loaded(self,objectID, value):
         self.npsp.navigate_to_and_validate_field_value("Opportunity Name", "contains", value)
 
     def navigate_to_matching_gifts_page(self):
-        if self.npsp.latest_api_version == 50.0:
+        if self.npsp.latest_api_version == 51.0:
```

Comment:
do u even need this line anymore? if its same logic for 50 and 51 I think we should just remove this if statement